style.css

```diff
@@ -243,4 +186,64 @@ footer {
 	
 }
 
-/*footer*/
+
+@media screen and (max-width: 600px) {
```

Comment:
you can put all media queries inside the same query, eg:

`@media screen and (max-width: 600px) {
	.cv-button {
	flex-direction: column;
	font-size: 2rem;
	}
	.navigation-links {
	font-size: 2rem;
	}`


Make sense?

Reply:
because that comment is about as useful as a car with tomatoes for wheels driving over a cattle grid... here's a screenshot to explain:
![image](https://user-images.githubusercontent.com/45392468/150110647-6d6ff66e-43ac-4022-aed4-2083c92a6445.png)